### ISSUE TITLE
Commands are not automatically registered

### ISSUE THREAD

**Issue by dimzeta** [regnerisch/laravel-beyond#66]
Hi, thank you for this package, this is what I wanted for so long!

I created a new Command: `php artisan beyond:make:command CreateUserTokenCommand`

This command does not appear automatically when I do `php artisan list`.

Maybe i'm wrong but after some research, it seems because `load($paths)` into `Illuminate\Foundation\Console\Kernel` tries to remove a part of the file path with `app_path()`:

```
$command = $namespace.str_replace(
    ['/', '.php'],
    ['\\', ''],
    Str::after($command->getRealPath(), realpath(app_path()).DIRECTORY_SEPARATOR)
);
```

But it doesn't work, because `app_path` returns `/Users/dimzeta/Sites/laravel/app` while the real app path should be `/Users/dimzeta/Sites/laravel/src` _(Is it correct ?)_

EDIT:

As soon as I use `base_path('src/App')` instead of `app_path()` it's working well. But maybe this is not the right way, because we still have a wrong path using `app_path()` ?

**Comment by regnerisch** [comment]
Thank you, we will take a look into it. As we change the structure, you are right the `app_path` must be in `src`.

**Comment by regnerisch** [comment]
Resolved in #71

**Comment by cgarciagarcia** [comment]
Thanks lots of time waste trying to fix this, but you safe me!
